symplyphysics/laws/kinematics/displacement_in_simple_harmonic_motion.py

```diff
@@ -7,80 +7,62 @@
 function of time.
 """
 
-from sympy import Eq, cos, symbols, dsolve, Function as SymFunction
+from sympy import Eq, cos, dsolve
 from symplyphysics import (
-    units,
     Quantity,
-    Symbol,
-    angle_type,
     validate_input,
+    symbols,
+    clone_as_symbol,
+    clone_as_function,
 )
 from symplyphysics.core.expr_comparisons import expr_equals
 from symplyphysics.core.quantity_decorator import validate_output_same
 from symplyphysics.core.symbols.quantities import scale_factor
 from symplyphysics.definitions import harmonic_oscillator_is_second_derivative_equation as harmonic_eqn
 
-displacement = symbols("displacement", real=True)
+displacement = clone_as_symbol(
+    symbols.any_dimension,
```

Comment:
My idea was to avoid the raw usage of the `SymbolNew` constructor (and the like) so that one would only need, in the ideal case, the `symbols` module and the `clone_as` family of functions. This would  In this case, `any_quantity` plays the role of a sort of a wildcard quantity that can represent any other one.

Reply:
Cloning any_quantity is ok.